src/blockchain_processor.py

```diff
@@ -348,6 +352,11 @@ def add_to_history(self, addr, tx_hash, tx_pos, tx_height):
         self.batch_txio[txo] = addr
 
 
+    def get_masternodes_status(self):
+        return self.dashd('masternode', ['list'])
+
+    def get_proposals_status(self):
+        return self.dashd('mnbudget', ['list'])
```

Comment:
@UdjinM6 If there's some info on which RPC methods to use depending on which version of dashd is running, I can probably have it choose the correct methods from the output of `getinfo`.